sass/elements/_details.scss

```diff
@@ -1,4 +1,64 @@
-details>summary {
-    // this should be the default
-    cursor: pointer;
+details {
+  $h-padding: 10px;
+  $border: 2px solid $subtitle-color;
+  // border: $border;
+  border-radius: $border-radius;
+  padding: 0 $h-padding;
+  margin-block: 1em;
+  background-color: rgba($color-white, 0.1);
+  overflow: hidden; // prevent clobbering corner radii
+
+  >summary {
+    cursor: pointer; // this should be the default
+    font-weight: bold;
```

Comment:
Given that now has a visually eye-catching background, I think this might be too "loud" as a default. Can we remove this in favor of the default boldness?

Reply:
sure; i waffled about it a few times myself.